packages/playground/src/index.ts

```diff
@@ -0,0 +1,283 @@
+import { getGentraceApiKey } from "@gentrace/core";
+import { WebSocket } from "ws";
+import { v4 as uuidv4 } from "uuid";
+
+export { init } from "@gentrace/core"; // for accessing the Gentrace API key
+
+import { AsyncLocalStorage } from "async_hooks";
+const asyncLocalStorage = new AsyncLocalStorage();
+
+interface customObject {
+  typeName: string;
+  objectName: string;
+  object: object;
+}
+
+interface interactionObject {
```

Comment:
General convention with type or interface definitions in TypeScript is to use [PascalCase](https://www.theserverside.com/definition/Pascal-case)

Reply:
[commit](https://github.com/gentrace/gentrace-node/pull/337/commits/4581b6aba21976ea82b4ab9415f9891a58c235c9)